### PR TITLE
Add Atom feed

### LINKED ISSUE #5: Add RSS feed

### PR DESCRIPTION
**Pull request by daveraine**
Closes #5

Went for Atom instead of RSS as it's newer / better. Who knew!

### CODE REVIEW

**Comment by mattdsmith89** [comment]
Looks 👍 to me

**Comment by daveraine** [comment]
I was gonna put the bulk of the code in a service class but it relies a bit on HttpContext so made sense to keep it all in the controller. Bit dirty but oh well.

File: src/Piston/AtomActionResult.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+﻿namespace Piston
```
Comment (daveraine) [inline src/Piston/AtomActionResult.cs]: Not sure if this class should go in a different namespace actually rather than the project root?

Comment (daveraine) [reply src/Piston/AtomActionResult.cs]: ❓ @mattdsmith89 @Kitcheone 😅

Comment (daveraine) [reply src/Piston/AtomActionResult.cs]: So looking at our favourite NuGetGallery they have [one custom action result in the project root](https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/ImageResult.cs) and [another in the Infrastrcture namespace](https://github.com/NuGet/NuGetGallery/blob/master/src/NuGetGallery/Infrastructure/SafeRedirectResult.cs). 

So do we go for Infrastucture? Leave where is? Something else? #bored

Comment (daveraine) [reply src/Piston/AtomActionResult.cs]: Moved it to infra... so it is ok? :shipit: ?

Comment (mattdsmith89) [reply src/Piston/AtomActionResult.cs]: Bit lonely in here 😮 looks good to me!

**Pull request merged**